app/src/org/commcare/android/resource/installers/ProfileAndroidInstaller.java

```diff
@@ -97,6 +100,24 @@ public boolean install(Resource r, ResourceLocation location, Reference ref,
         return false;
     }
 
+    private void checkAppTarget() throws UnfullfilledRequirementsException {
+        SharedPreferences prefs = CommCareApp.currentSandbox.getAppPreferences();
+        if (prefs.contains(KEY_TARGET_PACKAGE_ID)) {
+            String targetPackage = prefs.getString(KEY_TARGET_PACKAGE_ID, "");
+            if (!StringUtils.isEmpty(targetPackage)) {
+                String myAppPackage = CommCareApplication.instance().getPackageName();
+                if (!myAppPackage.contentEquals(targetPackage)) {
+                    String error = "This app requires " +
+                            (targetPackage.contentEquals("org.commcare.lts") ? "Regular Commcare (Non LTS) " : " Commcare LTS") +
```

Comment:
Isn't this backwards? If the `targetPackage` is set to LTS, then it should say "This app requires CommCare LTS"

Reply:
Ah silly! Thanks for catching this!